core-library/src/main/java/org/silverpeas/core/node/importexport/NodeTreeType.java

```diff
@@ -28,15 +28,28 @@
 
 import org.silverpeas.core.node.model.NodeDetail;
 
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
 /**
- * Classe utilisée pour le (un)marshalling Castor
+ * Classe utilisée pour le (un)marshalling
  * @author sdevolder
  */
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.NONE)
 public class NodeTreeType {
 
+  @XmlAttribute
   private String componentId;
+  @XmlElement(name = "topic")
   private NodeDetail nodeDetail;
 
+  public NodeTreeType() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1186)